### PR TITLE
overlord/fdestate: improve unit tests

### PR DESCRIPTION
**Pull request by bboozzoo**
Cherry picks of small unit tests improvements from DBX branch.

### CODE REVIEW

File: overlord/fdestate/fdemgr_test.go [hunk 1]
```diff
@@ -59,36 +91,40 @@ func (u *instrumentedUnlocker) Relock() {
...
 		return []byte{1, 2, 3, 4}, []byte{5, 6, 7, 8}, nil
 	})()
 
 	defer fdestate.MockVerifyPrimaryKeyHMAC(func(devicePath string, alg crypto.Hash, salt, digest []byte) (bool, error) {
-		c.Check(devicePath, Equals, "/dev/disk/by-uuid/bbb")
+		c.Assert(devicePath, Equals, "/dev/disk/by-uuid/bbb")
 		c.Check(alg, Equals, crypto.Hash(crypto.SHA256))
 		c.Check(salt, DeepEquals, []byte{1, 2, 3, 4})
 		c.Check(digest, DeepEquals, []byte{5, 6, 7, 8})
 		return true, nil
```
Comment (pedronis) [inline overlord/fdestate/fdemgr_test.go:116]: will we have a variant where this fails?

**Review by pedronis — APPROVED** [review]
thanks

**Review by ZeyadYasser — APPROVED** [review]
Thank you!

File: overlord/fdestate/fdemgr_test.go (hunk 1 above)
Comment (bboozzoo) [reply overlord/fdestate/fdemgr_test.go:116]: yes, there isn't one now, but it's TBD

**Pull request merged**